liquidator/OneInchExchange.js

```diff
@@ -0,0 +1,73 @@
+const Token = artifacts.require("ExpandedERC20");
+
+const oneSplitAbi = require("./abi/OneSplit.json");
+const oneSplitAddress = "0xC586BeF4a0992C495Cf22e1aeEE4E446CECDee0E";
+
+// As defined in 1inch
+const ETH_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
+
+class OneInchExchange {
+  /**
+   * @notice Creates OneInchExchange client
+   * @param {Object} web3 Web3 instance
+   * */
+  constructor({ web3, gasEstimator }) {
+    this.gasEstimator = gasEstimator;
+
+    this.web3 = web3;
+    this.web3.currentProvider.timeout = 1200000;
+    this.oneSplitContract = new web3.eth.Contract(oneSplitAbi, oneSplitAddress);
+
+    this.toBN = web3.utils.toBN;
+  }
+
+  /**
+   * @notice Swaps token on one inch
+   * @param {string} fromToken Address of token to swap from
+   * @param {string} toToken Address of token to swap to.
+   * @param {string} amountWei String amount to swap, in Wei.
+   * @param {Object} options Web3 options to supply to send, e.g.
```

Comment:
nit: shouldn't this just be two parameters -- one object that contains `fromToken`, `toToken`, and `amountWei` and then a separate argument for `options`?

Reply:
They're kind of logically seperated as the `options` param is the `options` when calling `web3.methods.<>.send({options})`

I was thinking the `from` and `value` field might be a bit confusing, so I separated them.

Reply:
Yep, totally makes sense -- this was just a comment on making sure the object structure was clear in the docstrings.